src/store-contract.spec.ts

```diff
@@ -136,7 +137,10 @@ describe("StoreContract", () => {
 				.should.eventually.be.rejectedWith(error);
 		});
 		it("should have at least 1 history", async () => {
-			await ctx.stub.getHistoryForKey("1001");
+			const history = await ctx.stub.getHistoryForKey("1001");
+			// Expected something like this, but undefined is being returned;
+			// expect(history).to.have.lengthOf(1);
```

Comment:
@yangricardo check this undefined return, I was expecting something with length/size of 1.